src/common/queries/useCoreApiInfo.ts

```diff
@@ -1,11 +1,11 @@
 import { useQuery } from '@tanstack/react-query';
 
-import { useApi } from '../api/useApi';
+import { useGlobalContext } from '../context/useGlobalContext';
 
 export function useCoreApiInfo() {
-  const api = useApi();
+  const { url: activeNetworkUrl } = useGlobalContext().activeNetwork;
   return useQuery({
     queryKey: ['coreApiInfo'],
-    queryFn: () => api.infoApi.getCoreApiInfo(),
+    queryFn: () => fetch(`${activeNetworkUrl}/v2/info`).then(res => res.json()),
```

Comment:
not using apiClient?

Reply:
this endpoint is not available through the existing library and will be added to a [Stacks RPC client](https://github.com/hirosystems/stacks-rpc-client) which is not production ready yet.